### PR TITLE
shortened URL for information retrieval book 

### PR DESCRIPTION
**Pull request by happiestbee**
## What does this PR do?
Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
please resolve conflicts.

**Comment by happiestbee** [comment]
I have resolved the commits, sorry for taking a while

File: books/free-programming-books-subjects.md [hunk 1]
```diff
@@ -670,7 +670,6 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 
 * [A short introduction to operating systems (2001)](http://markburgess.org/os/os.pdf) - Mark Burgess (PDF)
 * [Computer Science from the Bottom Up](http://www.bottomupcs.com)
-* [How to Make a Computer Operating System](https://github.com/SamyPesse/How-to-Make-a-Computer-Operating-System) (:construction: *in process*)
```
Comment (eshellman) [inline books/free-programming-books-subjects.md:673]: did you intend to remove this? if so, why?

Comment (happiestbee) [reply books/free-programming-books-subjects.md:673]: Oh, no I did not intent to remove that line sorry

**Comment by shrn01** [comment]
Existing URL works, so closing this PR

**Pull request closed without merging**